docs/articles/retrieval_augmented_generation_eval_qdrant_arize.md

```diff
@@ -220,21 +214,19 @@ Here's the aggregated view of our evaluations:
-
-In our Naive RAG evaluation, we get `Hallucination` = 18% and `QA Correctness` = 91%. Decent, but our aim is **0%** Hallucination and **100%** QA Correctness. So let's see if we can discover which query/ies lead to our less than ideal outcomes, and address the issue/s.
+Above, we can see `Total Traces` (essentially, the total number of eval queries run), `Total Tokens` processed, `Latency P50`, `Latency P99`, `Hallucination`, and `QA Correctness`, along with Overall `Relevance` metrics like *nDCG*, *Precision*, and *Recall*.
 
-For each eval question, `evaluations` captures the `Hallucination` and `QA Correctness` metrics for each query, making it easy to instantly spot problematic queries.
+In our Naive RAG evaluation, `Hallucination` = 18% and `QA Correctness` = 91%. Decent, but our aim is **0% Hallucination** and **100% QA Correctness**. So, let's see if we can figure out which query/ies lead to our less than ideal outcomes, and address the issue/s.
 
-We can zoom into each trace and inspect the spans therein to provide us with further details on each query and the output from the configured retriever.
+For each eval question, `evaluations` captures the `Hallucination` and `QA Correctness` metrics, making it easy to spot problematic queries immediately. We can zoom into each trace, and inspect its spans to get further details on each query and the output from the configured retriever.
 
 ![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/naive_rag_run_trace_details.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/naive_rag_run_trace_details.png)
 
-If we inspect the `Evaluations` tab, we’ll notice that the response was "hallucinated" even though it is "correct" according to QA Correctness metrics.
+If we take a look at the `Evaluations` tab, we can see that the response was "hallucinated" even though it is "correct" according to the QA Correctness metrics.
 
 ![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/naive_rag_run_trace_detail_eval.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/naive_rag_run_trace_detail_eval.png)
 
-This can happen for various reasons, including domain- or use case-specific terminology requiring exact matches. To handle this, we can enrich our retriever to be powered by **dense** as well as **sparse vectors**, that is, to **retrieve content based on semantic similarity as well as exact matches**.
+This can happen for various reasons, including domain- or use case-specific terminology requiring exact matches. To handle this, we can enrich our retriever so that it's powered by *not just sparse but also dense vectors*. That is, in addition to exact matches, we can **retrieve content based on semantic similarity**. We can build a hybrid RAG system.
```

Comment:
@robertdhayanturner Suggested change : 

This can happen for various reasons, including domain- or use case-specific terminology requiring exact matches. To handle this, we can enrich our retriever so that it's powered by *not just dense but also sparse vectors*. That is, in addition to **retrieve content based on semantic similarity** we can also cater to exact matches. We can build a hybrid RAG system.